extensions/amp-sidebar/0.1/amp-sidebar.css

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-sidebar {
+  position: fixed !important;
+  top: 0 !important;
+  max-height: 100vh !important;
+  height: 100vh;
+  max-width: 80vw !important;
+  background-color: #efefef;
+  min-width: 30vw !important;
+  overflow-x: hidden !important;
+  overflow-y: auto !important;
```

Comment:
Please create three bugs, assign to yourself and proceed at the earliest convenience (they will be launch blocking on this component):
 1. Possible integration with FixedLayer needed. This can be tested on iOS by scrolling and seeing if the `position:fixed` layout misbehaves. Also, other elements processed via FixedLayer can conflict with your logic, e.g. by being displayed above this menu.
 2. Check double-scrolling issue. Here, the test is on both iOS and Android. When scrolling the menu content, the main content should not scroll and vice-versa.
 3. Check the scrolling restoration issues on iOS and Android. E.g. open menu, scroll it and close. The main content should be unaffected by this.